eng/pipelines/build-windows-job.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       displayName: Build
       inputs:
         filePath: eng/build.ps1
-        arguments: -configuration ${{ parameters.configuration }} -prepareMachine -ci -build -publish -binaryLog -skipDocumentation ${{ parameters.buildArguments }}
+        arguments: -configuration ${{ parameters.configuration }} -prepareMachine -ci -build -binaryLog -skipDocumentation  /p:ProduceReferenceAssembly=false /p:NoPdbConversion=true /p:UsingToolVisualStudioIbcTraining=false /p:BuildingForLiveUnitTesting=true ${{ parameters.buildArguments }}
```

Comment:
What does `BuildingForLiveUnitTesting=true` do?

Reply:
re-doing this to make it clearer what each change is doing